.github/workflows/ci.yml

```diff
@@ -0,0 +1,37 @@
+name: CI
+
+on: [push]
+
+env:
+  CI: true
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        # Available OS's: https://help.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners
+        os: [ubuntu-20.04, windows-2019, macos-10.15]
+        node-version: [12.x, 10.x]
+    steps:
+    - uses: actions/checkout@v2.1.0
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1.4.1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: Cache node modules
+      uses: actions/cache@v1.1.2
+      env:
+        cache-name: cache-node-modules
+      with:
+        path: node_modules
+        key: ${{ runner.os }}-node${{ runner.node-version }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+    - run: npm ci
+    - run: npm run build
+    - run: npm test
+    - run: npx prettier --check "src/**"
```

Comment:
@NSeydoux I wasn't sure about this line - is all the relevant, Prettier'd source code in `src`?

Reply:
Some of it is under ``__tests__` as well, even if I'd like to move the test code alongside the tested files at some point.

Reply:
0fb275af6f2d8a8b381d2889d5212767143e9324